esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -1896,6 +1952,13 @@ void WsWuInfo::getWorkunitEclAgentLog(const char* fileName, const char* agentPid
     Owned<IFile> rFile = createIFile(fileName);
     if(!rFile)
         throw MakeStringException(ECLWATCH_CANNOT_OPEN_FILE, "Cannot open file %s.", fileName);
+
+    if (debugImportedStr.length()) //WU imported from a ZAP report
+    {
+        getWorkunitLogFromZAPFile(rFile, fileName, buf, outFile);
```

Comment:
can we switch from debugImportedStr to a flag like e.g. logsPerWorkunit,
and from getWorkunitLogFromZAPFile and similar to getWorkunitLogSingleFile

Reply:
We may switch from debugImportedStr to a flag like e.g. logsPerWorkunit if we do not need the flag to find out the log date and # of slaves from ZAP.

Reply:
Will do